libs/domain/site/dropdown-item/dropdown-item.component.ts

```diff
@@ -0,0 +1,36 @@
+import { ContentMixin } from '@spryker-oryx/experience';
+import { ButtonType } from '@spryker-oryx/ui/button';
+import { LitElement, TemplateResult, html } from 'lit';
+import { when } from 'lit/directives/when.js';
+import {
+  DropdownItemContent,
+  DropdownItemOptions,
+} from './dropdown-item.model';
+import { styles } from './dropdown-item.styles';
+
+export class DropdownItemComponent extends ContentMixin<
```

Comment:
I think we agreed with Helen it's a `MenuItemComponent`. @aleespryker can you double check with her?